app/src/main/java/net/squanchy/schedule/SchedulePageView.kt

```diff
@@ -0,0 +1,161 @@
+
+        tabLayout.addOnTabSelectedListener(TrackingOnTabSelectedListener(analytics, viewPagerAdapter))
+
+        setupToolbar()
+    }
+
+    private fun setupToolbar() {
+        val toolbar = findViewById<Toolbar>(R.id.toolbar)
+        toolbar.setTitle(R.string.activity_schedule)
+        toolbar.inflateMenu(R.menu.homepage)
+        toolbar.setOnMenuItemClickListener { item ->
+            when (item.itemId) {
+                R.id.action_search -> {
+                    navigate.toSearch()
+                    return@setOnMenuItemClickListener true
+                }
+                R.id.action_settings -> {
+                    navigate.toSettings()
+                    return@setOnMenuItemClickListener true
+                }
+                else -> return@setOnMenuItemClickListener false
+            }
+        }
+    }
+
+    override fun startLoading() {
+        subscription = service.schedule(false)
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe(
+                    {
+                        updateWith(
+                                it,
+                                object : ScheduleViewPagerAdapter.OnEventClickedListener {
+                                    override fun onEventClicked(event: Event) = onEventClickeds(event)
+                                }
+                        )
+                    },
+                    { Timber.e(it) }
+            )
+    }
+
+    private fun onEventClickeds(event: Event) {
+        analytics.trackItemSelected(ContentType.SCHEDULE_ITEM, event.id)
+        navigate.toEventDetails(event.id)
+    }
+
+    override fun stopLoading() {
+        subscription!!.dispose()
+    }
+
+    private fun hackToApplyTypefaces(tabLayout: TabLayout) {
+        // Unfortunately doing this the sensible way (in ScheduleViewPagerAdapter.getPageTitle())
+        // results in a bunch of other views on screen to stop drawing their text, for reasons only
+        // known to the Gods of Kobol. We can't theme things in the TabLayout either, as the
+        // TextAppearance is applied _after_ inflating the tab views, which means Calligraphy can't
+        // intercept that either. Sad panda.
+        tabLayout.addOnLayoutChangeListener { v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom ->
+            val context = tabLayout.context
+            val typeface = context.getFontFor(R.style.TextAppearance_Squanchy_Tab)
+
+            val tabCount = tabLayout.tabCount
+            for (i in 0 until tabCount) {
+                val tab = tabLayout.getTabAt(i)
+                if (tab == null || hasTypefaceSpan(tab.text)) {
+                    continue
+                }
+                tab.text = tab.text?.applyTypeface(typeface)
+            }
+        }
+    }
+
+    private fun hasTypefaceSpan(text: CharSequence?): Boolean {
+        return if (text !is Spanned) {
+            false
+        } else text.hasTypefaceSpan()
```

Comment:
Please add braces to the else too, and convert this to an expression form: `private fun hasTypefaceSpan(text: CharSequence?) = if () { } else { }`

Reply:
e93c5b1d3256962cc29adc8b3ac58e690303f32d